templates/controllers/grid/grid.tpl

```diff
@@ -83,7 +81,7 @@
 				so that we can restore it if the user deletes all rows.
 			**}
 			<tr>
-				<td colspan="{$grid->getColumnsCount('indent')|@count}">{translate key=$grid->getEmptyRowText()}</td>
```

Comment:
I believe that category grids have [their own template](https://github.com/pkp/pkp-lib/blob/master/templates/controllers/grid/gridBodyPartWithCategory.tpl#L30) where an empty `<tbody>` is created. This change adjusts the regular grids to match the category grid technique, but maybe I missed something.